admin/README.md

```diff
@@ -332,7 +332,6 @@ Class | Method | HTTP request | Description
  - [AWSInterfaceEndpoint](docs/AWSInterfaceEndpoint.md)
```

Comment:
@blva I still see a lot of breaking changes in the SDK related to removal of the classes. 
I need to be able to cross reference those with exceptions. Some of them were done by our team to fix CLI.

Do you know when we going to see release happening? I would like to be able to start testing breaking changes before GA